Adding lesspass and windscribe icons

andmagdo opened the pull request:
I have made these icons for lesspass (com.lesspass.android) and windscribe (com.windscribe.vpn).

For windscribe, I was unsure to use the provided fonts or to use the font that windscribe uses (as the w is the centerpiece of the icon), so I added both, marked with (Font W) for the one with the provided font, and marked (Default icon W) for the W in the style of the default windscribe icon.

As the commits show, I was using inkscape's svg optimizer and having issues with, but I fixed them (and the large volume of commits because I didn't commit locally).  As such, the svg files are pretty optimized, with no transform elements and one style tag

Donnnno commented:
The icons look good, windscribe already has an icon tho! (check the app)

maybe it just needs an appfilter, you can gather that with the icon request app!

andmagdo commented:
Lemme look for windscribe, one sec lol

andmagdo commented:
I see it, so how do I get the appfilter?

andmagdo commented:
Alright, got the appfilter (I would like to note that it would be helpful not to require a mail app--I hadn't set mine up yet)

it seems to be the same as line 13890 in /other/appfilter.xml
(com.windscribe.vpn/com.windscribe.vpn.splash.SplashActivity)

the email-to-be also included a link to grabbing it (which is incorrect, I got it from the aurora store, not f-droid, as it is not available on f-droid)

andmagdo commented:
The appfilter for lesspass is com.lesspass.android/com.lesspass.android.MainActivity

Donnnno commented:
> the email-to-be also included a link to grabbing it (which is incorrect, I got it from the aurora store, not f-droid, as it is not available on f-droid)

You can ignore the links.

No, I meant this icon request app, you can add the line of code easily in the appfilter.xml after that!
https://github.com/Kaiserdragon2/IconRequest/releases

https://github.com/Donnnno/Arcticons/blob/main/other/appfilter.xml

andmagdo commented:
Alright, I updated the appfilter list and deleted the excessive windscribe logos.

Is there anything else I should do (sorry for being a noob, I haven't dabbled in icons lol)
